carbonmark/components/pages/Retire/index.tsx

```diff
@@ -96,6 +104,45 @@ export const Retire: NextPage<PageProps> = (props) => {
             </div>
           </div>
         </div>
+
+        <div
+          className={cx(styles.fullWidth, {
+            whiteBG: isConnectedUser,
+          })}
+        >
+          <div className={styles.content}>
+            <div className={styles.cardsHeader}>
+              <Text t="h4" className={styles.textWithIcon}>
+                <Trans>Quick Retire</Trans>
+              </Text>
+            </div>
+
+            <Text className={styles.cardsDescription}>
+              <Trans>
+                Don’t want to go through the trouble of searching and selecting
+                a project to retire? Here’s a list of discount retirements from
+                trusted vendors.
+              </Trans>
+            </Text>
+
+            <div className={styles.cardsList}>
+              {props.defaultProjects.map((p) => {
+                if (p.isPoolProject)
+                  return (
+                    <ProjectCard
+                      key={p.id}
+                      project={p}
+                      url={createProjectPoolRetireLink(
+                        p,
+                        getPoolNameFromAddress(p.projectAddress) || "NBO" // typeguard, THE PROJECT ADDRESS FOR NBO 'VCS-981-2014'  IS NOT CORRECT ON CARBONMARK API !!
```

Comment:
pseudocode fix:
`p.prices.find(findNBO)`

Reply:
Thanks, I did it almost like this.